source/php/Module/views/button-template.blade.php

```diff
@@ -3,7 +3,7 @@
             @foreach($recommendLinkList as $recommendLink)
                 @include('partials.button', [
                     "text" => $recommendLink->recommendLinkLabel,
-                    "href" => $recommendLink->recommendLinkTarget,
+                    "href" => get_permalink($recommendLink->recommendLinkTarget),
```

Comment:
Functions in views should be avoided. Functionality should be placed in controller.